3d_segmentation/swin_unetr_btcv_segmentation_3d.ipynb

```diff
@@ -258,188 +282,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    " ## Download dataset and format in the folder.\n",
-    "    1. Download dataset from here: https://www.synapse.org/#!Synapse:syn3193805/wiki/89480\\n\n",
-    "    2. Put images in the ./data/imagesTr\n",
-    "    3. Put labels in the ./data/labelsTr\n",
-    "    4. make JSON file accordingly: ./data/dataset_0.json\n",
-    "    Example of JSON file:\n",
```

Comment:
Please do not delete this example JSON file, users can easily copy-paste and use it if they don't want to manually create a new one.

Thanks.

Reply:
I understand. I was trying to see if there is a more elegant way to do it, and I found one - I added a wget command that would directly get users the file into the data folder. Please let me know if that works. Thanks!